src/api/local-storage.js

```diff
@@ -0,0 +1,71 @@
+class localStorageApi {
+
+  static getUsers() {
+    const existingUsers = JSON.parse(localStorage.getItem("storedUser")) || [];
+    return existingUsers;
+  }
+
+  static getUser(userId) {
+    const existingUsers = localStorageApi.getUsers();
+    return existingUsers.find(storedUser =>  storedUser.id === userId );
```

Comment:
nit: could simplify even more
```suggestion
    return localStorageApi.getUsers().find(storedUser =>  storedUser.id === userId );
```
`getUsers()` returns an array, and you can pretty easily surmise from the function name you'll be getting some numbers of users, so just run the `.find` directly on the `getUsers` call.